guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -219,6 +219,161 @@ Create the host from an image with the `hammer host create` command and include
 
 For more information about additional host creation parameters for this compute resource, see xref:CLI_Params[].
 
+[[Provisioning_Virtual_Machines_in_VMware_vSphere-Provisioning_with_cloudinit_and_userdata_templates]]
+=== Using the VMware vSphere cloud-init and userdata Templates for Provisioning
+
+You can use VMware with the `Cloud-init` and `userdata` templates to insert user data into the new virtual machine, to make further VMware customization, and to enable the VMware-hosted virtual machine to call back to {Project} at intervals.
+
+You can use the same procedures to set up a VMware compute resource within {Project}, with a few modifications to the work flow.
+
+.VMware cloud-init Provisioning Overview
+
+When you set up the compute resource and images for VMware provisioning in {Project}, the following sequence of provisioning events occur:
+
+* The user provisions one or more virtual machines using the {Project} web UI, API, or hammer
+* {Project} calls the VMware vCenter to clone the virtual machine template
+* {Project} `userdata` provisioning template adds customized identity information
+* When provisioning completes, the {Project} `cloud-init` provisioning template instructs the virtual machine to call back to {SmartProxy} when `cloud-init` runs
+* VMware vCenter clones the template to the virtual machine
+* VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
+* The virtual machine builds, `cloud-init` is invoked and pings {Project} on port `80`, which then redirects to `443`
```

Comment:
It does not ping, this is a technical term for ICMP ping request. It simply "calls" back home using either HTTPS or HTTP depending on user configured in the template. The default is HTTPS (but certificate must be present).

Reply:
OK will change